lib/foreigner/schema_dumper.rb

```diff
@@ -42,6 +42,10 @@ def remove_prefix_and_suffix(table_name)
 
     def tables_with_foreign_keys(stream)
       tables_without_foreign_keys(stream)
+      # Ensure Foreigner to be initialized before running foreign_keys.
+      # This is required since schema::load is not initializing the environment
+      # anymore in Rails 4.1.9 (https://github.com/rails/rails/commit/5d6bb89f)
+      stream.puts '  Foreigner.load' if Module.const_defined?("Rails") && ::Rails.version == '4.1.9'
```

Comment:
Oops, that will not work. I am just thinking of a way to do this without the Rails reference (since the tests are broken with this addition)